x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/use_refresh_interval.ts

```diff
@@ -35,7 +37,9 @@ export const useRefreshInterval = (
     initAutoRefresh();
 
     function initAutoRefresh() {
-      const { value } = timefilter.getRefreshInterval();
+      const interval = globalState?.refreshInterval ?? timefilter.getRefreshInterval();
```

Comment:
If I open the Map view for a job after changing the refresh interval, and then switch back to the Jobs tab, the refresh interval goes back to the default of 30s:

![refresh_dfa_issue](https://user-images.githubusercontent.com/7405507/140977465-3822ae4a-9cf7-46af-8461-083ad20de8c9.gif)

Reply:
Good find @peteharverson - this will require some changes in the way we handle navigation here. Happy to do it in a follow up if that sounds alright.

Reply:
Thanks for taking a look @peteharverson 😄 
This has been fixed in 8f77b2ca8d43fe56cd54ccafdd03f89dbe5c81bb